TGC/decisions/SPA.txt

```diff
@@ -72,1738 +127,1066 @@ political_decisions = {
+			NOT = { has_country_flag = reconquest_lpl }
+			has_global_flag = reconquista
 		}
 
 		allow = {
 			war = no
+			is_greater_power = yes
+
 			OR = {
-				AND = {
-					nationalism_n_imperialism = 1
-					primary_culture = spanish
-				}
-				AND = {
-					any_owned_province = {
-						OR = {
-							continent = africa
-							continent = west_africa
-							continent = central_africa
-							continent = east_africa
-							continent = south_africa
+				ARG = {
+					OR = {
+						vassal_of = THIS
+
+						all_core = {
+							OR = {
+								owned_by = THIS
+
+								owner = {
+									vassal_of = THIS
+								}
+							}
 						}
-						is_colonial = no
-						NOT = { is_core = SAF }
 					}
-					naval_statistics = 1
 				}
-			}
-		}
 
-		effect = {
-			prestige = 4
-			1955 = { secede_province = THIS life_rating = 21 any_pop = { literacy = -0.99 } }
-			random_owned = {
-				limit = { owner = { NOT = { primary_culture = spanish } } }
-				any_country = {
-					limit = { primary_culture = spanish }
-					relation = { who = THIS value = -50 }
-					casus_belli = { target = THIS type = place_in_the_sun months = 36 }
-				}
-			}
-		}
+				ARC = {
+					OR = {
+						vassal_of = THIS
 
-		ai_will_do = { factor = 1 }
-	}
-
-	treaty_of_bata = {
-		picture = fernando_po
-		potential = {
-			owns = 1955
-			1973 = { empty = yes }
-			1976 = { empty = no }
-			1958 = { empty = no }
-			NOT = { has_country_flag = refused_treaty_on_bata }
-		}
-
-		allow = {
-			war = no
-		}
+						all_core = {
+							OR = {
+								owned_by = THIS
 
-		effect = {
-			random_owned = {
-				limit = {
-					owner = {
-						OR = {
-							owns = 1976
-							owns = 1958
+								owner = {
+									vassal_of = THIS
+								}
+							}
 						}
 					}
 				}
-				owner = { 1973 = { secede_province = THIS life_rating = 25 any_pop = { literacy = -0.99 } } }
-			}
-
-			random_owned = {
-				limit = { owner = { NOT = { owns = 1976 owns = 1958 } } }
-				owner = { 1976 = { owner = { country_event = 90141 } } }
 			}
 		}
 
-		ai_will_do = { factor = 1 }
-	}
+		effect = {
+			prestige = 20
 
-	#from Colonial Spain.txt
-	education_decree_1863 = {
-		picture = map_phillipines
-		potential = {
-			is_culture_group = iberian
-			OR = {
-				tag = SPA
-				tag = IBR
-				has_country_flag = is_latin_union
-			}
-			nationalism_n_imperialism = 1
-			has_country_flag = monarchy_gov
-			NOT = {
-				has_country_flag = education_decree_1863
-				any_owned_province = {
-					is_core = PHL
-					has_province_modifier = colonial_recruitment
+			any_country = {
+				limit = {
+					vassal_of = THIS
+					OR = {
+						tag = ARG
+						tag = ARC
+						tag = URU
+						tag = PRG
+					}
+				}
+				all_core = { add_core = VLP }
+				annex_to = THIS
+				relation = {
+					who = THIS
+					value = 400
 				}
 			}
-		}
 
-		allow = {
-			any_owned_province = { region = SPA_1455 }
-			invention = mission_to_civilize
-			money = 50000
-		}
+			URU = all_core = { add_core = VLP }
```

Comment:
Those two lines are incorrect, missing the outer brackets, causing a parse error. The correct code should go:
URU = { all_core = { add_core = VLP } }
PRG = { all_core = { add_core = VLP } }